docs/references/ruby/available-methods.mdx

```diff
@@ -13,7 +13,9 @@ All examples assume you have an instance of the `Clerk::SDK`:
 sdk = Clerk::SDK.new
 ```
 
-## [Allowlist identifiers](https://clerk.com/docs/reference/backend-api/tag/Allow-list-Block-list)
+## Allowlist identifiers
+
+The following methods are available for managing [allowlist identifiers](https://clerk.com/docs/reference/backend-api/tag/Allow-list-Block-list).
 
 Retrieve the list of allowlist identifiers.
```

Comment:
Is there a reason these aren't H3s?

Reply:
I think putting each method of an H3 makes sense since the page is specifically called "Available methods", and users will probably immediately look for what they want in the TOC

Reply:
So change all the method descriptions to h3's?